### PR TITLE
farmOS 3.3.0 blog post

### PR DESCRIPTION
**Pull request by mstenta**

### CODE REVIEW

File: posts/2024/farmOS-3.3.0/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+thanks to the maintainers of the
+[svelte-gantt](https://github.com/ANovokmet/svelte-gantt), which we used as
+the basis for farmOS-timeline.
+
+## Official ARM64 Docker Images
+
+We are now building official ARM64v7 and ARM64v8 Docker images and pushing them
+to the [farmOS Docker Hub repository](https://hub.docker.com/r/farmos/farmos).
+This means that it's easier to run farmOS in Docker on newer Raspberry Pi and
+Apple M1 processors. Previously, you would need to clone the farmOS Git
```
Comment (symbioquine) [inline posts/2024/farmOS-3.3.0/index.md]: > This means that it's easier to run farmOS in Docker on newer Raspberry Pi and Apple M1 processors.

Can somebody with an M# processor confirm this? I have been assuming this is true, but I don't have an M1 Mac to test with personally.

File: posts/2024/farmOS-3.3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+It is also now possible to export quantities from lists of logs. Logs are
+displayed in more places than quantities and provide different filtering
+options, and now a new "Export Quantity CSV" option is available on all log
+lists. This will export a CSV of all quantities that are associated with the
+selected logs.
+
+## Timeline library
+
+This release of farmOS includes a new experimental timeline rendering library
+that we have been experimenting with in the
```
Comment (symbioquine) [inline posts/2024/farmOS-3.3.0/index.md]: > experimental [...] experimenting with

Feels a bit redundant.

```suggestion
This release of farmOS includes a new experimental timeline rendering library
that we have begun leveraging in the
```

File: posts/2024/farmOS-3.3.0/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+[Crop Planning](https://drupal.org/project/farm_crop_plan) and
+[Grazing Planning](https://drupal.org/project/farm_crop_plan) modules. These
+modules (and others) can now leverage the new shared library to experiment with
+rendering timeline gantt visualizations. In the future, we hope to use the
+library in various places in the farmOS core UIs, as well.
```
Comment (symbioquine) [inline posts/2024/farmOS-3.3.0/index.md]:
```suggestion
[Grazing Planning](https://drupal.org/project/farm_crop_plan) modules. These
modules (and others) can now use the new shared library for
rendering timeline gantt visualizations. In the future, we hope to use the
library in additional farmOS core UIs as well.
```

File: posts/2024/farmOS-3.3.0/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+This library is being developed and maintained as a standalone project called
+[farmOS-timeline](https://github.com/farmOS/farmOS-timeline), similar to the
+existing [farmOS-map](https://github.com/farmOS/farmOS-map) library. Special
+thanks to the maintainers of the
+[svelte-gantt](https://github.com/ANovokmet/svelte-gantt), which we used as
```
Comment (symbioquine) [inline posts/2024/farmOS-3.3.0/index.md]:
```suggestion
thanks to the maintainers of
[svelte-gantt](https://github.com/ANovokmet/svelte-gantt), which we used as
```

**Review by symbioquine — CHANGES_REQUESTED** [review]
Mostly nitpicks

**Review by symbioquine — DISMISSED** [review]
LGTM!

File: posts/2024/farmOS-3.3.0/index.md (hunk 1 above)
Comment (paul121) [reply posts/2024/farmOS-3.3.0/index.md]: I believe Apple now has M1, M2, maybe M3 processors. Not sure the best way to generalize this language

Comment (mstenta) [reply posts/2024/farmOS-3.3.0/index.md]: Good point @paul121 - I changed "Apple M1" to "Apple M-series": https://github.com/farmOS/farmOS-community-blog/pull/29/commits/db48d62727ad24d0ef95f57c77f3a8c7ef4964d3

Comment (pcambra) [reply posts/2024/farmOS-3.3.0/index.md]: @symbioquine I've just tested in an M1 Pro and both 3.x and 3.3.0 seem to work correctly:

```
Digest: sha256:001c59a734f40840931ef7366f24b163006ff99ee8169933bbe856be234d2da9
Status: Downloaded newer image for farmos/farmos:3.3.0
docker.io/farmos/farmos:3.3.0

What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview farmos/farmos:3.3.0
```

```
Digest: sha256:962755e87ba77f02def18dd9310d364126ac7ea754057099be8c9aea292329b0
Status: Downloaded newer image for farmos/farmos:3.x
docker.io/farmos/farmos:3.x

What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview farmos/farmos:3.x
```

**Review by Farmer-Eds-Shed — APPROVED** [review]
LGTM

**Review by botlfarm — APPROVED** [review]
Looks good!

**Pull request merged**